README.rst

```diff
@@ -106,12 +106,13 @@ For more information about these options, see the `Getting Help`_ page.
 Issue Tracker
 *************
 
-We use JIRA for our issue tracker, not GitHub issues. You can search
-`previously reported issues`_.  If you need to report a problem,
+We use Github Issues for our issue tracker. You can search
+`previously reported issues`_.  If you need to report a bug, or want to discuss
+a new feature before you implement it, please `create a new issue`_.
 please make a free account on our JIRA and `create a new issue`_.
```

Comment:
I imagine you meant to delete this line.
```suggestion
```

Reply:
Oops. I meant to add a comment to the approval stating that you should make this fix and then it is approved. :)